README.md

```diff
@@ -133,6 +136,37 @@ for (var contractName in output.contracts['test.sol']) {
 }
 ```
 
+The ``smtSolver`` callback function is used to solve SMT queries generated by
+Solidity's SMTChecker.  If you have an SMT solver installed locally, it can
+be used to solve the given queries, where the callback must synchronously
+return either an error or the result from the solver.  A default
+``smtSolver`` callback is distributed by ``solc-js``, which relies on either
+Z3 or CVC4 being installed locally.
+
+#### Example usage with smtSolver callback
+
+```javascript
+var solc = require('solc');
+var smt = require('smtsolver');
+
+var input = {
+  language: 'Solidity',
+  sources: {
+    'test.sol': {
+      content: 'pragma experimental SMTChecker; contract C { function f(uint x) public { assert(x > 0); } }'
+    }
+  }
+};
+
+// New syntax (supported from 0.5.12, mandatory from 0.6.0)
```

Comment:
I would say to remove this otherwise it could confuse people into thinking SMT callback is supported <0.6.0